rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -165,10 +165,10 @@ if [ -n "${RRDCACHED_SERVER}" ]; then
 <?php
 \$config['rrdcached'] = "${RRDCACHED_SERVER}";
 EOL
-    cat >${LIBRENMS_PATH}/database/seeders/config/rrdcached.yaml <<EOL
+fi
+cat >${LIBRENMS_PATH}/database/seeders/config/rrdtool.yaml <<EOL
```

Comment:
I missed that in #225 but as I can see, this configuration is just for seeding, so one-time usage? What happens if somehow user remove the rrdtool version from the database and restart his container? Is the value still set? If not it should I think.

Reply:
This is a terrible example because this settings gets set automatically.
```sh
bash-5.1# lnms config:get rrdtool_version
1.7.2
bash-5.1# lnms config:set rrdtool_version

 Reset rrdtool_version to the default? (yes/no) [no]:
 > yes

bash-5.1# lnms config:get rrdtool_version
1.7.2
bash-5.1# 
```

Reply:
After initial config, the user is in charge of their settings.  That is the point of yaml config.

Without the yaml config, the user cannot disable services with `lnms config:set show_services 0` for example.

There are two settings that are locked in the config via php:
 1. update - because we don't want the install updating inside the container
 2. rrdached - because there exists an env variable to control this and we don't want to break that.
 
 I'm open for a discussion on if any other variables should be locked by setting them in php.

Reply:
Ok that makes sense!